libdeno/api.cc

```diff
@@ -30,12 +30,13 @@ Deno* deno_new(deno_buf snapshot, deno_buf shared, deno_recv_cb cb) {
   v8::Locker locker(isolate);
   v8::Isolate::Scope isolate_scope(isolate);
   {
+    v8::Local<v8::ObjectTemplate> global_template =
+        deno::CreateGlobalTemplate(isolate);
```

Comment:
This works. I would consider using our existing `deno::InitializeContext` for setting up the global variables... However using a ObjectTemplate is probably a better option - and how D8 is doing it - so don't worry about this nit for now.